core-bundle/src/Resources/contao/themes/flexible/src/basic.css

```diff
@@ -536,10 +540,15 @@ a.tl_submit {
 	width:100%;
 }
 .m12 {
-	margin:16px 15px 8px;
+	margin:0 15px;
+	padding:17px 0;
```

Comment:
I thought you wanted to revert the changes to the `m12` class so we do not have to add a `.cbx.m12` format definition?

Reply:
Yes, but I had to keep it, otherwise layout will break again.

Reply:
I see. 👍